syscall_darwin.go

```diff
@@ -18,3 +26,116 @@ func syscall_syscall9X(fn, a1, a2, a3, a4, a5, a6, a7, a8, a9 uintptr) (r1, r2,
 	callc(syscall9XABI0, unsafe.Pointer(&args))
 	return args.r1, args.r2, args.err
 }
+
+func NewCallback(fn interface{}) uintptr {
```

Comment:
Is it possible to dispose a created callback? We need a comment about the lifetime.